src/hooks/useDisableTableSortersIfLoading.ts

```diff
@@ -0,0 +1,21 @@
+import { useEffect } from "react";
+import { NetworkStatus } from "apollo-client";
+
+export const useDisableTableSortersIfLoading = (
+  networkStatus: NetworkStatus
+) => {
+  useEffect(() => {
+    const elements = document.querySelectorAll(
```

Comment:
I can see us wanting to make this more generalized in the near future, but this works for now

Reply:
i get what you mean but this is a hack we have to do to disable the ant table because it does not have the right API for us to do it via props. This is not the typical way to accomplish this kind of thing in React and will not be generalized.